recipe/conda_build_config.yaml

```diff
@@ -2,8 +2,3 @@ MACOSX_SDK_VERSION:        # [osx and x86_64]
   - "11.0"                 # [osx and x86_64]
 MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
   - "11.0"                 # [osx and x86_64]
-libevent:
-  - "2.1.10"
-  - "2.1.12"
```

Comment:
I saw #5, but the libevent migration has been [closed](https://github.com/conda-forge/conda-forge-pinning-feedstock/commit/33dddb36b3ae42d7fa02bcab8406eed78135f597) for 4 months; whatever external consumers of conda-forge packages must eventually move with the times.

Reply:
We know, we are currently trying to have them update their base environment to use the latest versions of the core dependencies such as `libevent`'s.